src/iterative_sorting/iterative_sorting.py

```diff
@@ -1,30 +1,59 @@
-# TO-DO: Complete the selection_sort() function below 
-def selection_sort( arr ):
+# TO-DO: Complete the selection_sort() function below
+def selection_sort(arr):
     # loop through n-1 elements
     for i in range(0, len(arr) - 1):
         cur_index = i
         smallest_index = cur_index
         # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
+        # (hint, can do in 3 loc)
+        for j in range(i + 1, len(arr)):
+            if arr[j] < arr[smallest_index]:
+                smallest_index = j
 
+        # TO-DO: swap
+        arr[cur_index], arr[smallest_index] = arr[smallest_index], arr[cur_index]
 
+    return arr
 
-        # TO-DO: swap
 
+# TO-DO:  implement the Bubble Sort function below
+def bubble_sort(arr):
```

Comment:
Not bubble sort..I think what you've come up with here is a version of selection sort.